File: python/ram.py
class RAM():
    #private members
    __capacity = 0
    __price = 0

    def __init__(self, capacity = 0, price = 0):
        self.__capacity = capacity
        self.__price = price

    #getter & setter
    def setcapacity(self, capacity):
        self.__capacity = capacity

    def getcapacity(self):
        return self.__capacity
    
    def setprice(self, price):
        self.__price = price

    def getprice(self):
        return self.__price

    def printRAM(self):
        print( "RAM         : %d GB" %(self.__capacity))    

File: prak3/ram.java
class RAM{
    //atribut
    private int capacity;
    private int price;
     
    //konstruktor 
    RAM(){}

    //setter & getter capacity
    public void setcapacity(int capacity){
        this.capacity = capacity;
    }
    public int getcapacity(){
        return capacity;
    }

    //setter & getter price
    public void setprice(int price){
        this.price = price;
    }
    public int getprice(){
        return price;
    }

    void printRAM(){
        System.out.println("RAM         : " + this.capacity + " GB");
    }
}